src/Features/Core/Portable/Shared/TestHooks/AsynchronousOperationListener.DiagnosticAsyncToken.cs

```diff
@@ -29,7 +29,7 @@ protected internal sealed class DiagnosticAsyncToken : AsyncToken
                 int lineNumber)
                 : base(listener)
             {
-                Name = Name;
+                Name = name;
```

Comment:
In case you were wondering who could make such a blatant typo ... it was me :frowning: